pkg/crc/machine/config/driver.go

```diff
@@ -15,4 +17,13 @@ func InitVMDriverFromMachineConfig(machineConfig MachineConfig, driver *drivers.
 	driver.BundleName = machineConfig.BundleName
 	driver.ImageSourcePath = machineConfig.ImageSourcePath
 	driver.ImageFormat = machineConfig.ImageFormat
+
+	for i, dir := range machineConfig.SharedDirs {
+		sharedDir := drivers.SharedDir{
+			Source: dir,
+			Target: dir,
+			Tag:    fmt.Sprintf("vol%d", i),
```

Comment:
Is it really a shared directory? it is mentioned here even that it is called a 'vol' (volume).

Reply:
`vol%d` is a totally arbitrary string, in this case I copied the naming from podman, but it can easily be changed to `dir%d` if you prefer.
Regarding the overall naming, https://virtio-fs.gitlab.io/ is mostly about "directory", and it's the same for https://libvirt.org/formatdomain.html#filesystems so 'directory' is fine in my opinion, and more specific than 'volume' . The macOS API is also about 'directories' https://developer.apple.com/documentation/virtualization/shared_directories